analyzer.go

```diff
@@ -175,13 +177,15 @@ func revertOriginalPathIn(text string, tmp string) string {
 // tryToSaveTo saves a file to given dir, preserving it's original path.
 // It only loggs any errors and does not fail. All files saved this way will
 // be in the root of the same dir.
-func tryToSaveTo(file *pb.File, tmp string) {
+func tryToSaveTo(file *pb.File, tmp string) (err error) {
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

Reply:
also now because it returns an error the comment is a little incorrect
And the name can be just `saveTo` now.
It's better to move logging out of this function too. Or it breaks recommendation "handle error only once". (sorry it's a bit too late and super quick google search didn't give me the result instantly, you can try to find it yourself :) )

Sorry for nit-picking.

Reply:
My fault, I didn't think about the doc. And you're right regarding handling logging the error outside given that currently returns the error.

> Sorry for nit-picking.

Not a problem at all!